packages/cf-deploy-config-inquirer/src/prompts/app-router-prompts.ts

```diff
@@ -100,6 +100,7 @@ function getRouterTypePrompt(): CfAppRouterDeployConfigQuestions {
         message: t('prompts.routerTypeMessage'),
         choices: [
             { name: t('routerType.managedAppRouter'), value: RouterModuleType.Managed },
+            { name: t('routerType.appFrontAppService'), value: RouterModuleType.AppFront },
```

Comment:
TSdoc needs to be updated with addition of AppFront

Reply:
Updated. Thanks.